app/services/gateway_list.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Validating available gateways
+class GatewayList < ApplicationService
+  GATEWAYS = %w[paypal stripe].freeze
+
+  def initialize(gateway)
+    @gateway = gateway
+  end
+
+  def call
+    validate_gateway
+  end
+
+  private
+
+  def validate_gateway
```

Comment:
Métodos que são apenas uma checagem e retornam booleanos normalmente a gente define com um ponto de interrogação no fim. Assim como o `.include?` que utilizou.
```suggestion
  def validate_gateway?
```